jsapp/js/enketoHandler.es6

```diff
@@ -67,9 +67,12 @@ const enketoHandler = {
             } else {
               let errorMsg = t('There was an error loading Enketo.');
               if (enketoData?.responseJSON?.detail) {
-                errorMsg += `<br><code>${enketoData.responseJSON.detail}</code>`;
+                // TODO: Format this to code with JSX with error template.
+                // Removed these tags because toast already escapes them.
+                //errorMsg += `<br><code>${enketoData.responseJSON.detail}</code>`;
+                errorMsg += enketoData.responseJSON.detail;
               }
-              notify(errorMsg, 'error');
+              notify.error(errorMsg);
```

Comment:
> `// TODO: format this code with JSX`

@duvld, I think I suggested we make this a todo because I was used to seeing .tsx extensions, and forgot that Babel jsx worked with .es6 files too.

Any other reason not to just to do it? 

> ```es6
> import React from 'react';
> ```

```es6
  // ...
  let errorMsg = (
    <>
      {t("There was an error loading Enketo.")}
      {enketoData?.responseJSON?.detail && (
        <>
          <br />
          <code>{enketoData.responseJSON.detail}</code>
        </>
      )}
    </>
  );
  notify.error(errorMsg);
  // ...
```